app/views/groups/_form.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= form_with(model: group) do |form| %>
+    <% if group.errors.any? %>
+      <div style="color: red">
+        <h2><%= pluralize(group.errors.count, "error") %> prohibited this group from being saved:</h2>
+        <ul>
+          <% group.errors.each do |error| %>
+            <li><%= error.full_message %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+    <div>
+      <%= form.label :name, style: "display: block" %>
+      <%= form.text_field :name %>
```

Comment:
- [ ] [OPTIONAL] Here we need an icon and not a user name to create a category, I strongly advise you to fix this issue.